sa/_db-next/migrations/20170718114448_DBCleanup.sql

```diff
@@ -0,0 +1,61 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+DROP TABLE externalCerts;
+
+DROP TABLE identifierData;
+
+ALTER TABLE certificateStatus DROP LockCol;
```

Comment:
Boulder won't use these because the `SetVersionCol` calls have been removed. This requires that the code is deployed before the migration but I think that is the default order anyway?